components/Home/HomeDownloadButton.tsx

```diff
@@ -14,15 +14,16 @@ const HomeDownloadButton: FC<NodeRelease> = ({
 }) => {
   const {
     frontMatter: { labels },
-  } = useNextraContext();
+  } = useLayoutContext();
 
   const { os, bitness } = useDetectOS();
 
   const nodeDownloadLink = downloadUrlByOS(versionWithPrefix, os, bitness);
   const nodeApiLink = `https://nodejs.org/dist/latest-v${major}.x/docs/api/`;
   const nodeAllDownloadsLink = `/download${isLts ? '/' : '/current'}`;
   const nodeDownloadTitle =
-    `${labels.download} ${version}` + ` ${labels[isLts ? 'lts' : 'current']}`;
+    `${labels?.download} ${version}` +
+    ` ${labels?.[isLts ? 'lts' : 'current']}`;
```

Comment:
nit: string interpolation and concatenation here. I could see it be purposeful for readability, but suggesting a change anyway to see what we prefer. I didn't check, perhaps it's a line length thing with prettier too. feel free to disregard if desired.

```suggestion
    `${labels?.download} ${version} ${labels?.[isLts ? 'lts' : 'current']}`;
```

Reply:
I would avoid making any further improvements to these files as they will be removed soon. They're legacy files.